exonum-java-binding/testkit/src/main/java/com/exonum/binding/testkit/UtcZonedDateTimeSerializer.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2019 The Exonum Team
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.exonum.binding.testkit;
+
+import static com.google.common.base.Preconditions.checkArgument;
+
+import com.exonum.binding.common.serialization.Serializer;
+import java.nio.ByteBuffer;
+import java.nio.ByteOrder;
+import java.time.Instant;
+import java.time.ZoneOffset;
+import java.time.ZonedDateTime;
+
+// TODO: duplicate of serializer in time-oracle - move to core? make it public in time-oracle?
```

Comment:
I think it is fine to make it public in _time-oracle_ with appropriate documentation (UTC-only, mostly for internal use, etc)